### PR TITLE
Now you can add images as UIImages

### PR DESCRIPTION
**Pull request by RamonGilabert**

### CODE REVIEW

File: Source/LightboxController.swift [hunk 1]
```diff
@@ -22,7 +22,7 @@ public class LightboxController: UIViewController {
     return manager
     }()
 
-  var images = [String]()
+  var images = [AnyObject]()
```
Comment (richardoti) [inline Source/LightboxController.swift]: Why not to store as NSURL by default?

**Comment by vadymmarkov** [comment]
@RamonGilabert I think it's better to avoid usage of AnyObject. We have initialiser that accepts strings, so we could create another init method that accepts array of images.

**Comment by RamonGilabert** [comment]
@vadymmarkov you meant like that?

**Comment by vadymmarkov** [comment]
@RamonGilabert Yeah, kind of :smiley:

**Pull request merged**